### PR TITLE
feat: upgrade go-restful v2 to go-restful v3

### LINKED ISSUE #620: Upgrade go-restful framework to v3
**What is the feature you want to add?**
Upgrade go-restful framework to v3
**Why do you want to add this feature?**
Currently, we use go-restful v2, with no more updates basically. the main version is v3. more feature.
e.g. v2 only supports swagger 1.2. but v3 supports swagger 2.0, in the future supports openapi 3.0.
**How to implement this feature?**

**Additional context**

### PR DESCRIPTION
**Pull request by onecer**
**Please provide issue(s) of this PR:**
Fixes #620 

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Configuration
- [ ] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Naming
- [ ] HealthCheck
- [ ] Test and Release

**Please check any characteristics that apply to this pull request.**

- [X] Does not have any user-facing changes. This may include API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Review by chuntaojun — APPROVED** [review]

File: apiserver/grpcserver/base.go [hunk 1]
```diff
@@ -20,13 +20,15 @@ package grpcserver
...
 	"context"
 	"fmt"
-	"github.com/polarismesh/polaris-server/common/secure"
-	"google.golang.org/grpc/credentials"
 	"net"
 	"net/http"
 	"strings"
 	"time"
 
+	"google.golang.org/grpc/credentials"
```
Comment (houseme) [inline apiserver/grpcserver/base.go:28]: 按组import 同下

File: apiserver/eurekaserver/applications_test.go [hunk 2]
```diff
@@ -27,10 +27,12 @@ import (
 	"github.com/polarismesh/polaris-server/service"
 
 	"github.com/golang/protobuf/ptypes/wrappers"
+
 	api "github.com/polarismesh/polaris-server/common/api/v1"
```
Comment (houseme) [inline apiserver/eurekaserver/applications_test.go:31]: 顺带整理一下 import

File: apiserver/eurekaserver/server.go [hunk 3]
```diff
@@ -21,14 +21,15 @@ import (
...
 	"fmt"
-	"github.com/polarismesh/polaris-server/common/secure"
 	"net"
 	"net/http"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/emicklei/go-restful"
+	"github.com/polarismesh/polaris-server/common/secure"
```
Comment (houseme) [inline apiserver/eurekaserver/server.go:30]: 顺带整理一下 import

File: apiserver/eurekaserver/applications_test.go (hunk 2 above)
Comment (onecer) [reply apiserver/eurekaserver/applications_test.go]: 已处理

File: apiserver/eurekaserver/server.go (hunk 3 above)
Comment (onecer) [reply apiserver/eurekaserver/server.go]: 已处理

**Review by chuntaojun — APPROVED** [review]

**Review by houseme — APPROVED** [review]
LGTM

**Comment by onecer** [comment]
处理imports 分组

```
find . -name "*.go" -type f |grep -v .pb.go|xargs -I {} goimports-reviser -rm-unused -format {} -project-name github.com/polarismesh/polaris-server && find . -name "*.go" -type f |grep -v .pb.go|grep -v test/tools/tools.go|xargs -I {} goimports  -local github.com/polarismesh/polaris-server  -w {}
```

**Pull request merged**